src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/RazorFileChangeKind.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin
+{
+    public enum RazorFileChangeKind
+    {
+        Added,
+        Removed,
+        Saved,
```

Comment:
Might as well name it changed to keep things consistent. E.g, what if you switch a git branch, the file might change even though we aren't technically saving.